R/httrWrappers.R

```diff
@@ -0,0 +1,78 @@
+# Internal wrappers for httr request functions 
+# 
+# The primary reason for this wrapper is to add the appropriate authorization header to requests.
+# This file also centralizes http error handling.
+GET <- function(url, config = list(), ..., handle = NULL) {
```

Comment:
maybe we use .GET, .POST, .PUT instead of just GET, POST, PUT

to prevent potential name conflict with HTTR::GET, HTTR:POST

Reply:
Yes I like this idea. I'll make the change.